shell-maker.el

```diff
@@ -97,6 +97,11 @@ For example:
   :type 'directory
   :group 'shell-maker)
 
+(defcustom shell-maker-forget-file-after-clean nil
+  "Tell the `shell-maker-save-session-transcript' reset file path or not."
```

Comment:
nit: Consider "If non-nil, reset file path after clear command." as other booleans in this file.

ps. Heads-up, I've just made https://github.com/xenodium/chatgpt-shell/commit/54f31661df1c44e16638c8ff1e457573ea3ef7cd which is more idiomatic.